fxa-auth-db-server/docs/DB_API.md

```diff
@@ -249,7 +250,7 @@ Parameters.
 
 Each token takes the following fields for it's create method respectively:
 
-* sessionToken : data, uid, createdAt
+* sessionToken : data, uid, createdAt, userAgent, os, deviceType
```

Comment:
> Naming is hard. @philbooth let's try to chat about this early next week before I break for PTO.

That would be really good if we could, as there's some other things I'm uncertain about here too. For the record:

1. `deviceType` is not currently the same as what we discussed briefly last week. Instead of a generic `mobile|desktop` classification, it is a more specific `iPhone|Blackberry` kind of thing which would perhaps be better characterised as `deviceName`. However, we could still achieve the generic device type that we wanted by using a different user-agent parser. That would mean using a different dependency to the content server though, and I'm not sure whether that harmony between repos is important to us.

2. I'm not currently recording any version information because instinctively it didn't feel that useful, especially for browsers with rapid release cycles. Was that the right or wrong thing to do?

3. I briefly wondered about normalising this information in a separate table. On the one hand, I imagine we're going to end up with a lot of duplicates which could just be identical foreign keys. On the other hand, it's an extra `JOIN` when we fetch the data and conceptually they're not exactly related and it's not much data anyway so what's the point, really?